packs/data/spell-effects.db/spell-effect-wish-twisted-form-critical-failure.json

```diff
@@ -0,0 +1,99 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 5
+        },
+        "rules": [
+            {
+                "key": "Weakness",
+                "type": "{item|flags.pf2e.rulesSelections.spellEffectWishTwistedFormFailure}",
+                "value": "{item|system.level.value}"
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Wish-Twisted Form",
+                "selector": "speed",
+                "type": "status",
+                "value": -10
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Wish-Twisted Form",
+                "selector": "saving-throw",
+                "type": "status",
+                "value": -1
+            },
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Damage.RollFlavor.acid",
+                        "value": "acid"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.cold",
+                        "value": "cold"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.electricity",
+                        "value": "electricity"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.fire",
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.force",
+                        "value": "force"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.negative",
+                        "value": "negative"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.positive",
+                        "value": "positive"
+                    },
+                    {
+                        "label": "PF2E.Damage.RollFlavor.sonic",
+                        "value": "acid"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "Choose an energy damage type"
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "ac",
+                "type": "status",
+                "value": -1
+            }
+        ],
+        "source": {
+            "value": ""
```

Comment:
"value": "Pathfinder Advanced Player's Guide"

Reply:
Is there a way to edit this from within Foundry? I don't mind editing JSON but it's a little cumbersome.

Reply:
Unfortunately there's no UI for it yet.